hardware/raspberrypi/conformity.md

```diff
@@ -0,0 +1,19 @@
+# Standards Conformance
+
+Raspberry Pi models conform to various standards. Those documents published are linked below.
+
+## Compute Module 3 and Compute Module 3 Lite
+
+All Raspberry Pi Compute Module 3 devices and the IO board have undergone extensive compliance testing and meet the following standards.
+
+- EU
+  - Electromagnetic Compatibility Directive (EMC) 2014/30/EU,
+  - Restriction of Hazardous Substances (RoHS) Directive 2011/65/EU
+  - EN55032:2012 Class B
+
+- USA
+  - ANSI C63.4 
+  - FCC 47 CFR Part 15, Subpart B, Class B Digital Device.
+
+ [FCC Conformance Document](https://www.raspberrypi.org/files/compliance/RaspberryPiCM3_DOC_FCC.pdf)   
+ [EU conformance Document](https://www.raspberrypi.org/files/compliance/RaspberryPiCM3_DOC_FCC.pdf)
```

Comment:
These both link to the same PDF - is that correct?
Perhaps the second link should be to https://www.raspberrypi.org/files/compliance/RaspberryPiCM3_DOC_EU.pdf